install.rdf

```diff
@@ -8,11 +8,17 @@
     <em:name>Quicktext</em:name>
     <em:version>1.1</em:version>
     <em:unpack>true</em:unpack>
-    <em:description>Adds a toolbar with unlimited number of text to quickly insert. It's also possible to use variables like [[TO=firstname]]. With settings for everything.</em:description>
-    <em:creator>Emil Hesslow</em:creator>
+    <em:description>Adds a toolbar with unlimited number of text to quickly insert. It’s also possible to use variables like [[TO=firstname]]. With settings for everything.</em:description>
     <em:contributor>R Kent James</em:contributor>
-    <em:contributor>John Bieling</em:contributor>    
-    <em:translator>Óvári (hu)</em:translator>    
+    <em:contributor>John Bieling</em:contributor>
+    <em:contributor>Philippe Lieser</em:contributor>
+    <em:translator>(de)</em:translator>
+    <em:translator>Emil Hesslow (en-US)</em:translator>
+    <em:translator>(fr)</em:translator>
+    <em:translator>Óvári (hu)</em:translator>
+    <em:translator>(ja)</em:translator>
+    <em:translator>(sv-SE)</em:translator>
+    <em:creator>Emil Hesslow</em:creator>
```

Comment:
is there a reason, you moved the creator to the bottom?

Reply:
Copied what you did in your TBSync add-on

Reply:
But this is Emil, I would like to have him at the top as before.

Reply:
done